lmdeploy/model.py

```diff
@@ -353,6 +353,8 @@ def match(cls, model_path: str) -> Optional[str]:
         if 'llava' in path and 'v1' in path and 'v1.6-34b' not in path \
             and 'mistral' not in path:
             return 'llava-v1'
+        elif 'llava-1.5' in path:
```

Comment:
Doesn't llava 1.5 use `vicuna`?

Reply:
llava 1.5 uses `vicuna` and it's name is [llava-hf/llava-1.5-7b-hf](https://huggingface.co/llava-hf/llava-1.5-7b-hf), which does not have `v1` in model path.